dll/win32/oleaut32/oleaut.c

```diff
@@ -873,7 +874,7 @@ static HRESULT reg_get_typelib_module(REFIID iid, WCHAR *module, DWORD len)
 
     RegCloseKey(ikey);
 
-    sprintf(typelibkey, "Typelib\\%s\\%s\\0\\win%u", tlguid, ver, sizeof(void *) == 8 ? 64 : 32);
+    StringCbPrintfA(typelibkey, sizeof(typelibkey), "Typelib\\%s\\%s\\0\\win%u", tlguid, ver, sizeof(void *) == 8 ? 64 : 32);
```

Comment:
That's Wine code. You should surround it with `#if __REACTOS__`

Reply:
done